.github/workflows/frontend-shared.yml

```diff
@@ -0,0 +1,34 @@
+name: frontend-shared package
+on:
+  release:
+    types:
+      - published
+jobs:
+  update-frontend-shared-version:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    # Setup .npmrc file to publish to npm
+    - name: Setup Nodejs
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+        registry-url: 'https://registry.npmjs.org'
+    - name: Install client
```

Comment:
An install of the client is needed for the imports to work in the .github/actions/.../index.js file(s).

Because we're already installing the client, it made more sense to bundle the package from the client (using `yarn setup-frontend-shared`) rather than changing into the frontend-shared dir and running `npm run build` 
That means the `build` command in frontend-shared will likely be unused until the whole package moves into its own repository.


`yarn setup-frontend-shared and`
vs
`cd setup-frontend-shared && npm run build`

run on completely different code paths but should produce the same result.